jekyll/_cci2/bitbucket-integration.adoc

```diff
@@ -150,9 +150,9 @@ After the necessary permissions have been set up, the next step is adding a `.ci
 
 After you create and commit a `.circleci/config.yml` file to your Bitbucket repository, CircleCI immediately checks your code out and runs your first job along with any configured tests.
 
-CircleCI runs your tests on a clean container every time so that your tests are fresh each time you push code, and so that your code is never accessible to other users. Watch your tests update in real-time on https://circleci.com/dashboard[your dashboard]. You can also get status updates through email notifications, or look for the status badges that appear on Bitbucket. Integrated statuses also appear on the pull request screen, to show that all tests have passed.
+CircleCI runs your tests on a clean container every time so that your tests are fresh each time you push code, and so that your code is never accessible to other users. Watch your tests update in real-time on link:https://circleci.com/dashboard[your dashboard]. You can also get status updates through email notifications, or look for the status badges that appear on Bitbucket. Integrated statuses also appear on the pull request screen, to show that all tests have passed.
 
-See the <<config-intro#,Configuration Tutorial>> page for a configuration walkthrough.
+See the xref:config-intro#[Configuration tutorial] for a configuration walkthrough.
```

Comment:
```suggestion
See the xref:config-intro[Configuration tutorial] for a configuration walkthrough.
```

Reply:
Works fine with this left in soo you can ignore if you prefer

Reply:
Yeah I started adding the # because without it I noticed it was working inconsistently